integrations-and-supported-tools/bokeh/scripts/requirements.txt

```diff
@@ -0,0 +1,2 @@
+bokeh
```

Comment:
@SiddhantSadangi the requirements here should have explicit versions so that people know what works.
We'll be updating versions periodically.